### ISSUE TITLE
InputNode : Get image sequences

### ISSUE THREAD

**Issue by Tilix4** [eliemichel/CodenameGogh#39]
- [ ] Get image sequence with file selector.
- [ ] Parse image name structure to extract FFMPEG (%XXd) command.

**Comment by eliemichel** [comment]
We could also support the `"/path/to/filename.###.png"`` syntax which is pretty common for file sequence handling.

**Comment by Tilix4** [comment]
Yes, that was my point. Convert the "filename.###.png" pattern to "filename.%03d"  ffmpeg command.
